lib/code_corps/github/events/installation.ex

```diff
@@ -13,5 +24,89 @@ defmodule CodeCorps.GitHub.Events.Installation do
   - do the work
   - marked the passed in event as "processed" or "errored"
   """
-  def handle(%GithubEvent{}, %{}), do: :not_fully_implemented
+  @spec handle(GithubEvent.t, map) :: {:ok, GithubEvent.t}
+  def handle(%GithubEvent{action: "created"} = event, payload) do
+    event
+    |> start_event_processing()
+    |> do_handle(payload)
+    |> stop_event_processing(event)
```

Comment:
The point of the `processing` state here was that we could, in the UI, at least redirect the user to the proper project, although we could not present them yet with the list of GitHubRepos. Since I think the most common case in terms of timing upon redirect is "we've received the webhook and we're processing it", that means keeping them off the interstitial page and into the least surprising UI, which I think is preferable. It will make channels easier for us to deal with in that event, too.